lib/Github/Api/PullRequest.php

```diff
@@ -163,6 +158,10 @@ public function merge($username, $repository, $id, $message, $sha, $mergeMethod
             $mergeMethod = $mergeMethod ? 'squash' : 'merge';
         }
 
+        if (!in_array($mergeMethod, array('squash', 'rebase'))) {
```

Comment:
That's fine but there are several other places throughout the codebase that behave in this way already, just do a search for in_array to see what I mean.  Should an issue be opened to remove all of those checks in the next major version for consistency?